renovate.json

```diff
@@ -0,0 +1,134 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": [
+		"config:base",
+		"helpers:pinGitHubActionDigests",
+		":dependencyDashboard",
+		":semanticCommits",
+		":gitSignOff"
+	],
+	"timezone": "Europe/Vienna",
+	"schedule": [
+		"before 5am on wednesday"
+	],
+	"labels": [
+		"dependencies",
+		"3 - to review"
+	],
+	"commitMessageAction": "Bump",
+	"commitMessageTopic": "{{depName}}",
+	"commitMessageExtra": "from {{currentVersion}} to {{#if isPinDigest}}{{{newDigestShort}}}{{else}}{{#if isMajor}}{{prettyNewMajor}}{{else}}{{#if isSingleVersion}}{{prettyNewVersion}}{{else}}{{#if newValue}}{{{newValue}}}{{else}}{{{newDigestShort}}}{{/if}}{{/if}}{{/if}}{{/if}}",
+	"rangeStrategy": "bump",
+	"rebaseWhen": "conflicted",
+	"ignoreUnstable": false,
+	"baseBranches": [
+		"main",
+		"stable4.4",
+		"stable4.3"
```

Comment:
This means updating dependencies all the way down to Nextcloud 25. Is this ok? I noticed in mail we still update stable1.15 which corresponds to 24 (and lower).

@ChristophWurst

Reply:
AFAIK, this only applies to vulnerability alerts though.

Reply:
yes only for vulns and even that is broken https://github.com/renovatebot/renovate/discussions/20542#discussioncomment-5389609